### PR TITLE
certify: bump deps

### PR DESCRIPTION
**Pull request by OndrejSlamecka**
Specify library name and version:  **certify/cci.20201114**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
@OndrejSlamecka Thank you for your contribution, please, ask for authorization at the issue #4 as described on the comment above. Otherwise, the CI will not build your PR.

File: recipes/certify/all/conanfile.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.82.0")
+        self.requires("boost/[>=1.82 <2]")
```
Comment (uilianries) [inline recipes/certify/all/conanfile.py:41]:
```suggestion
        self.requires("boost/1.83.0")
```

Boost is not allowed to use ranged versions, please, keep the latest version instead.

More info: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Comment (OndrejSlamecka) [reply recipes/certify/all/conanfile.py]: Thanks!

**Review by uilianries — APPROVED** [review]
LGTM. Just need to wait for allowed users list now. Thank you for your PR!

**Review by jwillikers — APPROVED** [review]

**Comment by OndrejSlamecka** [comment]
This has been obsoleted by https://github.com/conan-io/conan-center-index/pull/21192

**Pull request closed without merging**